.npmrc

```diff
@@ -0,0 +1,3 @@
+//registry.npmjs.org/:_authToken=${NPM_TOKEN}
+registry=https://registry.npmjs.org/
+always-auth=true
```

Comment:
This file isn't necessary if `NPM_TOKEN` is defined. Plus, while probably rare, it could override a registry mapping if someone uses a passthrough registry.

Reply:
I tried to get rid of it but it complained and said it was not logged in

Reply:
I have a _.npmrc_ file in my home directory to stay logged in, but it doesn't need to be part of the project.